easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11.1-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11.1'
```

Comment:
@hajgato: this version is not available at http://xcb.freedesktop.org/dist/ ? Only `1.11` is?